docs/Debugging.md

```diff
@@ -11,7 +11,7 @@ next: testing
 To access the in-app developer menu:
 
 1. On iOS shake the device or press `control + ⌘ + z` in the simulator.
-2. On Android shake the device or press hardware menu button (available on older devices and in most of the emulators, e.g. in [genymotion](https://www.genymotion.com) you can press `⌘ + m` to simulate hardware menu button click)
+2. On Android shake the device or press hardware menu button (available on older devices and in most of the emulators, e.g. in [genymotion](https://www.genymotion.com) you can press `⌘ + m` to simulate hardware menu button click).  You can also install [Frappé](http://getfrappe.com), a tool for OS X to shake devices remotely and binds CMD+SHIFT+R hotkey.
```

Comment:
"which binds to CMD+SHIFT+R"?

I don't know how to parse the sentence: "a tool for OS X to shake devices remotely and binds CMD+SHIFT+R hotkey".